src/front/mod.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg(feature = "glsl")]
+pub mod glsl;
```

Comment:
just a thought: if the preprocessor has a lot of complexity and code size, it needs be optional. I'm sure many users will have their own preprocessors anyway (i.e. for `include` handling)